webpack/webpack.dev.js

```diff
@@ -21,9 +21,7 @@ module.exports = {
             options: {
               sourceMap: true,
               localsConvention: 'camelCase',
-              modules: {
-                localIdentName: '[local]___[hash:base64:5]'
-              }
+              modules: false
```

Comment:
I wonder why did you choose to go this way

Reply:
So that I can have a separate folder for styles with variables, shared, etc, that don't have to be imported in .scss file where I use them, but in one place - the general.scss

Reply:
Yeah :) the question is why do you want all that  :) At the end of the day, that is up to you

Reply:
If I import lets say variables.scss on top of every file, wouldn't that contribute to a huge css output? Because it will import it n number of times instead of just one?

Reply:
emmmmm, are you sure?

Reply:
not anymore :) I'll look into it